app/view_models/auction_show_view_model.rb

```diff
@@ -65,7 +65,10 @@ def bid_status_presenter
   end
 
   def bid_label
-    if over? && auction.bids.any?
+    if available?
+      bidding_status_presenter.bid_label(current_user)
+    # will replace below in future issues
```

Comment:
@harrisj do you mind removing this comment? I think an issue would be more helpful